lib/rule_engine/ast.py

```diff
@@ -237,6 +238,19 @@ def from_string(cls, context, string):
 			dt = dt.replace(tzinfo=context.default_timezone)
 		return cls(context, dt)
 
+class TimedeltaExpression(LiteralExpressionBase):
+	"""
+	Literal datetime expressions representing a specific point in time. This expression type always evaluates to true.
```

Comment:
```suggestion
	Literal timedelta expression representing an offset from a specific point in time. This expression type always evaluates to true.
```

Reply:
Thanks, this was an oversight. I'll get it fixed up 👍